ui/models.py

```diff
@@ -20,6 +20,16 @@
 TRANSCODE_PREFIX = 'transcoded'
 
 
+class Channel(models.Model):
+    """
+    Model for Video Channels
+    """
+    title = models.TextField()
+    description = models.TextField(null=True, blank=True)
+    slug = models.UUIDField(unique=True, null=False, blank=False, default=uuid4)
+    owners = models.ManyToManyField(settings.AUTH_USER_MODEL)
```

Comment:
Any advantage of this approach vs having one creator/owner and using moira_lists to determine who can administer/edit/view?